commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/portlet/action/EditCommerceAccountMVCActionCommand.java

```diff
@@ -154,12 +154,24 @@ protected String getSaveAndContinueRedirect(
 	protected void setActive(ActionRequest actionRequest)
 		throws PortalException {
 
+		long[] commerceAccountIds = null;
+
 		long commerceAccountId = ParamUtil.getLong(
 			actionRequest, "commerceAccountId");
 
+		if (commerceAccountId > 0) {
+			commerceAccountIds = new long[] {commerceAccountId};
+		}
+		else {
+			commerceAccountIds = StringUtil.split(
+				ParamUtil.getString(actionRequest, "commerceAccountIds"), 0L);
```

Comment:
This should be `ParamUtil.getLongValues` rather than `StringUtil.split(ParamUtil.getString...`